### ISSUE TITLE
cx:chartSpace is Missing

### ISSUE THREAD

**Issue by ScottHaney** [OfficeDev/Open-XML-SDK#675]
**cx:chartSpace is Missing**

When trying to create a histogram chart (an example of an extended chart) in office the cx:chartSpace xml element is required. A C# class for this exists in an uncompiled test which is named `DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing.ChartSpace`. For some reason this class isn't included in the sdk.

**Information**

- .NET Framework
- DocumentFormat.OpenXml Version: 2.10

**Repro**

```csharp
DrawingsPart drawingsPart = ...;
var extendedChartPart = drawingsPart.AddNewPart<ExtendedChartPart>();

\\I want the below line to work, but the ChartSpace element doesn't exist. Also it would be nice to be able to call extendedChartPart.ChartSpace = rather than extendedChartPart.RootElement =
extendedChartPart.RootElement = new DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing.ChartSpace();
``` 

**Observed**

It doesn't compile and there doesn't seem to be an easy workaround

**Comment by twsouthwick** [comment]
Thanks @ScottHaney for the report. Any of the new Office 2016 related schema entries were added as best effort knowing that there were some gaps, but figured getting what we had out would be better than postponing until we had full support. Seems like you found one of the gaps.

@tomjebo This looks like it may be related to the issue you are already investigating.

**Comment by ScottHaney** [comment]
@twosouthwick - I don't mind doing some updates to the code to help with the missing features. I am in need of them to get the histogram plot working. How do I update the code to add the missing items? For instance in the `Chart` class with `SchemaAttr` value 80 there is an `Id` attribute missing. If I just add a property named `Id` to `Chart` with an `Index` attribute it won't serialize out the `Id` value to the xml when the document is saved. Where in the code base do you go to make changes to the generated files and have those changes show up in the xml produced?

**Comment by twsouthwick** [comment]
Unfortunately, the source has a code generator we can't make public (hopefully someday). I've been working towards removing those magic numbers, but it's a slow process... right now, you can't just add things out. @tomjebo has found the issue and is looking to submit a fix.

**Comment by ScottHaney** [comment]
@twsouthwick - Thanks for the response! I found that I was able to use the public method `OpenXmlElement.SetAttribute` to add attributes that would serialize out. Also in the meantime I added a custom implementation of the `ExtendedChartPart` which allows me to hack my way into creating histograms finally.

**Comment by twsouthwick** [comment]
FYI there's a fix for this in #687 that @tomjebo is working through

**Comment by tomjebo** [comment]
@ScottHaney #687 was merged, can you please retest and let me know if you're not able to get the histogram chart working?
